universal-login-relayer/test/core/services/EncryptedWalletHandler.test.ts

```diff
@@ -0,0 +1,46 @@
+import {getKnexConfig} from '../../testhelpers/knex';
+import {EmailConfirmationsStore} from '../../../src/integration/sql/services/EmailConfirmationsStore';
+import {EncryptedWalletsStore} from '../../../src/integration/sql/services/EncryptedWalletsStore';
+import {EncryptedWalletHandler} from '../../../src/core/services/EncryptedWalletHandler';
+import {EmailConfirmation, TEST_ENCRYPTED_WALLET_JSON, EncryptedWallet} from '@unilogin/commons';
+import {expect} from 'chai';
+
+describe('INT: EncryptedWalletHandler', () => {
+  const notConfirmedEmail = 'example@email.com';
+  const confirmedEmail = 'confirmed@email.com';
+  let encryptedWalletHandler: EncryptedWalletHandler;
+  let encryptedWallet: EncryptedWallet;
+
+  before(() => {
+    const knex = getKnexConfig();
```

Comment:
destroy knex instance at the end of test

Reply:
Done